std_msgs/Header header
ClusterBounds[] clusters
